cypress/support/step_definitions/manualAccountCreation/manualAccountCreation.ts

```diff
@@ -581,7 +581,7 @@ When('{string} is verified as grey', (addContactDetailsButton: string) => {
   cy.get('#submitForm').should('be.enabled', addContactDetailsButton);
 });
 
-Then('I verify the status of {string} to {string}', (linkText: string, status: string) => {
+Then('I see the status of {string} to {string}', (linkText: string, status: string) => {
```

Comment:
Should be 'I see the status of {string} is {string}' for the regex

Reply:
need more clarity on this